chia/rpc/wallet_rpc_api.py

```diff
@@ -433,6 +454,10 @@ async def create_new_wallet(self, request: Dict):
                     "backup_dids": did_wallet.did_info.backup_ids,
                     "num_verifications_required": did_wallet.did_info.num_of_backup_ids_needed,
                 }
```

Comment:
This is a long function so I added 3 explicit `passes`. 
Maybe it can be packaged differently.